### ISSUE TITLE
Auto Bot Connect!  [Connect Button]

### ISSUE THREAD

**Issue by Numan1982** [dahquan/rattlesnake#79]
as by the title i can make bots connect to a new server without restarting the console. i did this by a connect button were u have to click it to connect. but i added a little feauture were if u click stop connection it stops the console without having it to close and when u press connect again the bots connection again to the new server.

**Comment by WJKH** [comment]
And... your issue is?
Please don't open issues to comment us you have done something.

If you want contribute to the project just make a PR.

**Comment by ravyn120** [comment]
That is very easy to make... Like really the simplest way is to use websockets or ws and use the server to emit data to the user and back so when it receives do ws.close or open kinda like socket.io

**Comment by ravyn120** [comment]
@Numan1982 Is yours like this? https://gyazo.com/41917e19b36249ac87449f6e8c6d06b3 I made that one like a few days ago. I'll prob PR this

**Comment by yario-o** [comment]
@ravyn120 I have exactly the same button :)

**Comment by anonymousxxx02** [comment]
@ravyn120 can you help me do that... iam using @l3mpik script

**Comment by ravyn120** [comment]
@yario-o really? I never copied anyone's code meh I used socket.io anyway

**Comment by yario-o** [comment]
@ravyn120 yes I know but I have exactly the same button (With same size and same color)

**Comment by SIrStarch** [comment]
i am abit confused ive done everything but node doesnt start up the feedme.js and i see nowhere to login

**Comment by Numan1982** [comment]
ravyn i aint using ur script.
